cmake/modules/BuildPlugin.cmake

```diff
@@ -30,6 +30,8 @@ MACRO(BUILD_PLUGIN PLUGIN_NAME)
 		ADD_GEN_QRC(RCC_OUT "${PLUGIN_NAME}.qrc" PREFIX artwork/${PLUGIN_NAME} ${PLUGIN_EMBEDDED_RESOURCES})
 	ENDIF(ER_LEN)
 
+	SET(CMAKE_AUTOMOC ON)
+	SET(CMAKE_AUTOUIC ON)
```

Comment:
These were already turned on in the base `CMakeLists.txt`, this would be where we'd add the:
```cmake
set_target_properties("${PLUGIN_NAME}" PROPERTIES AUTOUIC_SEARCH_PATHS ***figure_out_the_path_somehow_here***)
```

Reply:
Thanks for the hint. Will look.

Reply:
nope, doesn't work

Reply:
You need to add this line after the target has been defined (the `add_library` command).

Reply:
I have tried. Let's check now.

Reply:
Looks like it doesn't work

Reply:
Looked into the logs again, should i turn on autouic manually?

Reply:
I think so, yes. The other place it's turned on is in `src/CMakeLists.txt`, so it is scoped to the `src` directory and won't affect plugins. I wouldn't expect this line here to be needed at all - the `.ui` files are in the same directory as the headers that reference them, so CMake should be able to find it without any hints.

Reply:
At last, this has been fixed.